internal/engine/exit/actions.go

```diff
@@ -1,8 +1,40 @@
 package exit
 
-type Action struct {
-	ExitSignal bool
-	ExitError  error
+import (
+	"errors"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/tilt-dev/tilt/internal/store"
+	tiltruns "github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
```

Comment:
Ha no that was just a slippery finger, will make it singular.

FWIW I sorta stole this convention from K8s stuff I saw (maybe kubebuilder book?) and I _believe_ the reason for it is to avoid an obnoxiously large diff when the entity version changes - just the import alias + affected logic will be in the diff that way.

I know Nick has done something similar (from a limiting diff blast radius perspective) of adding a type **alias**, which has an added benefit of cutting out redundancy when referring to the type since it gives a local name for it.